packages/nextjs/app/builders/[0xC4Aad525854Cc8d21122Bee8CcC1d9c3cEcBb859]/page.tsx

```diff
@@ -0,0 +1,90 @@
+"use client";
```

Comment:
We don't need this restriction since your page doesn't use any client-only mechanism.

Please remove this line so we can improve the app's performance by pre-rendering the page during the compile process.